### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

Add sentence to documentation about TTL renewal

- [ ] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.10: *(Please link PR)*
  - [ ] Backport for 3.9: *(Please link PR)*
  - [ ] Backport for 3.8: *(Please link PR)*

#### Related Information

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

**Review by dothebart — APPROVED** [review]
LGTM, thanks.

File: Documentation/DocuBlocks/Rest/Queries/post_api_cursor_cursor.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ still return no documents. If, however, *hasMore* is *false*, then
 the cursor is exhausted.  Once the *hasMore* attribute has a value of
 *false*, the client can stop.
 
+If the cursor is not fully consumed, the time-to-live for the cursor will
+be renewed by this API call.
```
Comment (nerpaula) [inline Documentation/DocuBlocks/Rest/Queries/post_api_cursor_cursor.md:34]:
```suggestion
If the cursor is not fully consumed, the time-to-live for the cursor
is renewed by this API call.
```

File: Documentation/DocuBlocks/Rest/Queries/post_api_cursor_cursor_batch.md [hunk 2]
```diff
@@ -36,6 +36,8 @@ application, it should explicitly delete the cursor to inform the server that it
 successfully received and processed the batch so that the server can free up
 resources.
 
+The time-to-live for the cursor will be renewed by this API call.
```
Comment (nerpaula) [inline Documentation/DocuBlocks/Rest/Queries/post_api_cursor_cursor_batch.md:39]:
```suggestion
The time-to-live for the cursor is renewed by this API call.
```

File: Documentation/DocuBlocks/Rest/Queries/put_api_cursor_cursor.md [hunk 3]
```diff
@@ -35,6 +35,9 @@ still return no documents. If, however, *hasMore* is *false*, then
 the cursor is exhausted.  Once the *hasMore* attribute has a value of
 *false*, the client can stop.
 
+If the cursor is not fully consumed, the time-to-live for the cursor will
+be renewed by this API call.
```
Comment (nerpaula) [inline Documentation/DocuBlocks/Rest/Queries/put_api_cursor_cursor.md:39]:
```suggestion
If the cursor is not fully consumed, the time-to-live for the cursor
is renewed by this API call.
```

**Pull request merged**